Sources/StreamChatUI/Components.swift

```diff
@@ -130,8 +130,69 @@ public struct _Components<ExtraData: ExtraDataTypes> {
     public var channelReadStatusView: ChatChannelReadStatusCheckmarkView.Type =
         ChatChannelReadStatusCheckmarkView.self
 
+    // MARK: - Message composer components
+
+    /// The view controller used to compose a message.
+    public var messageComposerVC: _ComposerVC<ExtraData>.Type =
+        _ComposerVC<ExtraData>.self
```

Comment:
It seems to me a good idea to have VC suffixes for controllers in `Component` as well. 
@nuno-vieira @VojtaStavik WDYT?

Reply:
Yes that is the aim 👍